edgegrid/signer.go

```diff
@@ -32,6 +33,26 @@ func AddRequestHeader(config Config, req *http.Request) *http.Request {
 		req.Header.Set("Content-Type", "application/json")
 	}
 
+	_, AkamaiCliEnvOK := os.LookupEnv("AKAMAI_CLI")
+	AkamaiCliVersionEnv, AkamaiCliVersionEnvOK := os.LookupEnv("AKAMAI_CLI_VERSION")
+	AkamaiCliCommandEnv, AkamaiCliCommandEnvOK := os.LookupEnv("AKAMAI_CLI_COMMAND_VERSION")
```

Comment:
This one is looking for the wrong ENV car, should be `AKAMAI_CLI_COMMAND` not `AKAMAI_CLI_COMMAND_VERSION`